src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -156,6 +156,32 @@ public IActionResult DrainStatus([FromServices] IScriptHostManager scriptHostMan
             }
         }
 
+        [HttpPost]
+        [Route("admin/host/resume")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        public async Task<IActionResult> Resume([FromServices] IScriptHostManager scriptHostManager)
+        {
+            _logger.LogDebug($"Received request to resume a draining host - host status: {scriptHostManager?.State.ToString()}");
+
+            if (!Utility.TryGetHostService(scriptHostManager, out IDrainModeManager drainModeManager)
+                || scriptHostManager?.State != ScriptHostState.Running)
+            {
+                _logger.LogWarning("The host is not in a state where we can resume.");
```

Comment:
Thinking this should be Debug rather than Warning, otherwise this may generate noise the user can't control and they may question?